circle.yml

```diff
@@ -9,10 +6,21 @@ machine:
 dependencies:
   pre:
     - sudo service docker stop; sudo curl -L -o /usr/bin/docker 'http://s3-external-1.amazonaws.com/circle-downloads/docker-1.6.0-circleci'; sudo chmod 0755 /usr/bin/docker; sudo service docker start; true
+  cache_directories:
+    - unrejector/vendor/bundle
+  override:
+    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3:
+        pwd: unrejector
+
+test:
+  override:
+    - ./meta/bin/check_markdown_links
+    - bundle exec rspec --color --require spec_helper spec --format progress:
+        pwd: unrejector
 
 deployment:
   hub:
     branch: master
     commands:
       - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - git clone git://github.com/hackedu/hack-camp /tmp/hack-camp && cd /tmp/hack-camp/unrejector && make push
```

Comment:
Also, just for reference, but newlines at the end of files is part of POSIX. See http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline